priv/repo/migrations/20251022103623_create_ai_comments.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Cadet.Repo.Migrations.CreateAiCommentLogs do
+  use Ecto.Migration
+
+  def change do
+    create table(:ai_comment_logs) do
+      add(:submission_id, references(:submissions, on_delete: :delete_all), null: false)
+      add(:question_id, references(:questions, on_delete: :delete_all), null: false)
+      add(:raw_prompt, :text, null: false)
+      add(:answers_json, :text, null: false)
+      add(:response, :text)
+      add(:error, :text)
+      add(:comment_chosen, {:array, :text})
+      add(:final_comment, :text)
+      timestamps()
+    end
+
+    create(index(:ai_comment_logs, [:submission_id]))
+    create(index(:ai_comment_logs, [:question_id]))
```

Comment:
Are these indices strictly necessary? Since AI comments (+ logs nonetheless) are very often being written (and in massive volumes), adding an index will result in significantly degraded write performance. FK lookup is already really well-optimized on Postgres, I don't see a need for the index imo.